Examples/AdvancedImageReading/Documentation.rst

```diff
@@ -0,0 +1,60 @@
+.. _lbl_advanced_image_reading:
+
+Advanced Image Reading
+==============================
+
+Overview
+--------
+
+This example illustrates advanced image reading options:
+  1. Explicitly selecting a specific IO for reading.
+  2. Querying an image for its meta-data without reading the bulk intensity data.
+  3. Reading a sub-region of an image.
+
+Selecting a Specific IO
+++++++++++++++++++++++++
+When using the default settings for the `ImageFileReader class <https://itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1ImageFileReader.html>`_ or
```

Comment:
The seletecting an imageIO should be a separate example from the read meta-data and sub-region example.